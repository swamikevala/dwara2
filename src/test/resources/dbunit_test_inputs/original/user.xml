<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <user id="21001" hash="$2a$10$70nZ.1zvmmgAXQZ5qDFHxe08eTijEejJ5HRZAtwRcPuMjw4MfRley" name="pgurumurthy" priorityband_id="6001"/>
  <user id="21002" hash="$2a$10$pfrQKh7LEw3oklfohO/WoOb0yUjL6pLAkmKQLDbA6tDO2BcZ.Eon." name="apr"/>
  <user id="21003" hash="$2a$10$JDBuv5l9F1jJEOk8JqGQVOoqxGRaRNIujWIK.7y9jXUh99S65xEr." name="Sadhguru"/>
  <action_user permission_level="2" action_id="8003" user_id="21001"/>
  <libraryclass_action_user action_id="8001" libraryclass_id="5001" user_id="21001"/>
  <libraryclass_action_user action_id="8002" libraryclass_id="5001" user_id="21001"/>
  <libraryclass_action_user action_id="8003" libraryclass_id="5001" user_id="21001"/>
</dataset>
