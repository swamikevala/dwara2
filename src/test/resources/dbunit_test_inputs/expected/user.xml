<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <user id="21001" name="pgurumurthy"/>
  <user id="21002" name="apr"/>
  <user id="21003" name="Sadhguru"/>
  <user id="21004" name="ShivaShambho"/>
</dataset>
